test/Kestrel.Transport.Libuv.Tests/LibuvTransportTests.cs

```diff
@@ -128,6 +131,12 @@ public class LibuvTransportTests
             }
 
             await transport.UnbindAsync();
+
+            if (!await serviceContext.ConnectionManager.CloseAllConnectionsAsync(default).ConfigureAwait(false))
```

Comment:
So this is what was causing test errors?

Reply:
Yep